src/form-field/form-field.svelte

```diff
@@ -0,0 +1,49 @@
+<script>
+  import classes from '../_utils/classes.js';
+
+  let _class = null;
+  export { _class as class };
+  export let nameClass = null;
+  export let helpClass = null;
+
+  export let name = null;
+  export let help = null;
+
+  export let required = false;
+  export let optional = false;
+  export let errors = null;
+</script>
+
+<div class={classes('form-field', _class)}>
+  <slot name="description">
+    <div class="description">
+      {#if name != null}
+        <div class={classes('name', nameClass)}>
+          {name}
+          {#if required}
+            <span class="required">*</span>
+          {/if}
+        </div>
+      {/if}
+      {#if help != null}
+        <div class={classes('help', helpClass)}>{help}</div>
+      {/if}
+    </div>
+  </slot>
+  <div class="field">
+    <slot />
+    {#if errors != null}
+      {#each errors as error}
+        {#if error}<div class="message error">{error}</div>{/if}
+      {/each}
+    {/if}
+    {#if required}
+      <div class="message info">* Required</div>
+    {:else if optional}
+      <div class="message info">Optional</div>
+    {/if}
```

Comment:
Use slots with default for easier customization and i18n

Reply:
Tried putting something together about this, realized that I don't like any of it. Custom messages can be passed using a slot below, and as for i18n, we can simply wrap the text in a i18n mapper later on in the library. Don't feel like having those slots here.